chainercv/links/model/pspnet/pspnet.py

```diff
@@ -0,0 +1,476 @@
+from __future__ import division
+
+from math import ceil
+import warnings
+
+import chainer
+import chainer.functions as F
+import chainer.links as L
+from chainercv.utils import download_model
+import numpy as np
+import six
+
+try:
+    from chainermn.links import MultiNodeBatchNormalization
+except Exception:
+    warnings.warn('To perform batch normalization with multiple GPUs or '
```

Comment:
Instead of raising a warning during import, it should be raised when PSPNet is instantiated.

Reply:
Please fix this. Please make sure that the error is raised only when `comm` is specified.

Also, we should install the latest release.